diff --git a/src/sshguard.in b/src/sshguard.in
index 74e28cc..2cecf1e 100644
--- a/src/sshguard.in
+++ b/src/sshguard.in
@@ -5,7 +5,7 @@
 # entire process group (subshell) on exit/interrupts.
 trap "trap - SIGTERM && kill 0" SIGINT SIGTERM EXIT
 
-libexec="@libexecdir@"
+libexec="$SNAP/@libexecdir@"
 version="@sshguardversion@"
 
 err() {
@@ -28,7 +28,7 @@ EOF
 }
 
 # Source configuration file
-config="@sysconfdir@/sshguard.conf"
+config="$SNAP_DATA/sshguard.conf"
 if [ ! -r $config ]; then
     err "Could not read '$config'"
     err "Please configure SSHGuard."
